auth-api/src/auth_api/resources/entity.py

```diff
@@ -144,13 +148,20 @@ class ContactResource(Resource):
     @staticmethod
     @_jwt.requires_auth
     @TRACER.trace()
+    @user_context
     @cors.crossdomain(origin='*')
-    def get(business_identifier):
+    def get(business_identifier, **kwargs):
         """Get contact email for the Entity identified by the provided business identifier."""
+        user_is_staff = False
```

Comment:
This isn't needed

Reply:
This function throws a 404_NOT_FOUND error without the staff bypass

Reply:
That's fine, we don't use the data from it anyways